datastream-server-api/src/main/java/com/linkedin/datastream/server/api/strategy/AssignmentStrategy.java

```diff
@@ -48,4 +49,21 @@
    */
   Map<String, Set<DatastreamTask>> assign(List<DatastreamGroup> datastreams, List<String> instances,
       Map<String, Set<DatastreamTask>> currentAssignment);
+
+
+  /**
+   * Assign partition for a paritulcar datastream group to all the tasks in current assignment
+   * It return a map from instance -> tasks map with partition info stored in the task
+   * This interface needs to implemented if the Brooklin Coordinator is going to perform the
+   * partition assignment.
+   *
+   *
+   * @param currentAssignment the old assignment
```

Comment:
Can you amend the documentation to clarify whether this is current assignment for the datastreamGroup or for all datastreams?

Reply:
Its for all the datastreamgroup. I will update the doc. Thanks